github/repos_hooks.go

```diff
@@ -98,6 +98,31 @@ type createHookRequest struct {
 	Active *bool                  `json:"active,omitempty"`
 }
 
+// PubSubHubBubRequest represents the request to be send
```

Comment:
Let's please change this to `SubscribePubSubHubbubOpts`  (Note the lower-case "bub").

```
SubscribePubSubHubbubOpts represents the options to be sent for the SubscribePubSubHubbub endpoint.
```